manifests/config.pp

```diff
@@ -10,13 +10,87 @@
     system => true,
   }
 
+  case $::osfamily {
```

Comment:
please don't use those legacy facts anymore. Instead use `$facts['os']['name']`

Reply:
thanks! Fixed in 2e3af3a4f789bd6d8813dea0fa43de6c2cc243ca